qiita_pet/uimodules/prep_template_tab.py

```diff
@@ -161,12 +163,17 @@ def render(self, study, prep_template, full_access, ena_terms,
         if raw_data_id:
             rd = RawData(raw_data_id)
             rd_ft = rd.filetype
+
             # If the prep template has a raw data associated, it can be
             # preprocessed. Retrieve the pre-processing parameters
             if rd_ft in ('SFF', 'FASTA'):
                 param_iter = Preprocessed454Params.iter()
             elif rd_ft == 'FASTQ':
-                param_iter = PreprocessedIlluminaParams.iter()
+                param_iter = [pip for pip in PreprocessedIlluminaParams.iter()
```

Comment:
This test shouldn't be against `pip.barcode_type != 'not-barcoded'`? The reason is that if we then add more non-barcode options here (for multiple parameters sweeping) we will need to list them by name in here.